Presentation.Web/app/components/it-contract/it-contract-plan.controller.ts

```diff
@@ -822,7 +824,23 @@
                                 operator: "contains"
                             }
                         }
-                    }
+                    },
+                    {
+                        field: "LastChangedName", title: "Sidst redigeret: Bruger", width: 150,
+                        persistId: "lastchangedname",
+                        template: dataItem => `${dataItem.LastChangedByUser.Name} ${dataItem.LastChangedByUser.LastName}`,
+                        hidden: true,
+                        sortable: false,
+                        filterable: false
+                    },
+                    {
+                        field: "LastChanged", title: "Sidste redigeret: Dato", width: 150,
+                        persistId: "lastchanged",
```

Comment:
make sure this is actually interpreted as a dat input when filteren.. again see in tisystem catalog